### ISSUE TITLE
`vue-i18n-routing` now integrated in `@nuxtjs/i18n`

### ISSUE THREAD

**Issue by razorness** [victorgarciaesgi/nuxt-typed-router#139]
**Describe the bug**
`@nuxtjs/i18n` is not depending on `vue-i18n-routing` anymore. So, `nuxt-typed-router` generated types do not in match with `@nuxtjs/i18n` anymore. See https://github.com/nuxt-modules/i18n/commit/7a0a9bd2697abf3143e72b33ecff978dee0aad48

After upgrade to current `@nuxtjs/i18n`, you get this:

```
 ERROR  Pre-transform error: Failed to resolve import "vue-i18n-routing" from ".nuxt/typed-router/__i18n-router.ts". Does the file exist?
```

A workaround is to add `vue-i18n-routing` as dev dependency and hope for duck typing.


**Environnement infos**

```
Nuxt project info:                                                                                                                                                                                                                                            1:59:34 PM  

------------------------------
- Operating System: Linux
- Node Version:     v20.11.0
- Nuxt Version:     3.10.2
- CLI Version:      3.10.1
- Nitro Version:    2.8.1
- Package Manager:  yarn@4.1.0
- Builder:          -
- User Config:      site, gtag, runtimeConfig, imports, srcDir, serverDir, modules, plugins, css, postcss, sitemap, robots, schemaOrg, pinia, build, vite, alias, typescript, devtools
- Runtime Modules:  nuxt-typed-router@3.5.1, @nuxtjs/i18n@8.1.1, @nuxtjs/tailwindcss@6.11.4, @nuxtjs/apollo@5.0.0-alpha.11, v-wave/nuxt, @pinia/nuxt@0.5.1, @nuxtjs/eslint-module@4.1.0, @nuxt/image@1.3.0, nuxt-seo-experiments@4.0.0-rc.4, nuxt-simple-robots@4.0.0-rc.14, @nuxtjs/sitemap@5.1.0, nuxt-og-image@3.0.0-rc.38, nuxt-schema-org@3.3.4, nuxt-gtag@1.2.0
- Build Modules:    -
------------------------------
```

**Comment by victorgarciaesgi** [comment]
Thanks for the report, i'll update this

**Comment by victorgarciaesgi** [comment]
Seems like they removed options from `useLocalePath` and `useLocaleRoute`. Will maybe be a breaking change for me too

**Comment by victorgarciaesgi** [comment]
Fixed in `3.6.0`
